### PR TITLE
Add emails for wg-etcd-operator leads on sigs list

### PR DESCRIPTION
**Pull request by ahrtr**
Refer to https://groups.google.com/a/kubernetes.io/g/leads/c/3A2d1_onI3o

cc @hakman @jberkus @jmhbnz @justinsb

### CODE REVIEW

**Comment by ahrtr** [comment]
cc @BenTheElder

**Comment by jmhbnz** [comment]
/hold
Hold until everyone has confirmed

**Review by jmhbnz — COMMENTED** [review]
/lgtm

**Comment by hakman** [comment]
I can confirm that @justinsb's email is also ok.

File: sigs.yaml [hunk 1]
```diff
@@ -3450,18 +3450,23 @@ workinggroups:
...
       company: VMware
+      email: benjamin.ahrtr@gmail.com
     - github: hakman
       name: Ciprian Hacman
       company: Microsoft
+      email: ciprianhacman@gmail.com
     - github: jberkus
       name: Josh Berkus
       company: Red Hat
+      email: josh@agliodbs.com
```
[the remark this answers is not in the document]
Comment (ahrtr) [reply sigs.yaml]: Done.

I got your email `josh@agliodbs.com` from the share list of [this doc](https://docs.google.com/document/d/1Tk3XS75fprI-iBPk7BkjS7ezZDjWSk-zfhPfX9T2dAc/edit?tab=t.0#heading=h.ol96621u8gmq)

File: sigs.yaml [hunk 2]
```diff
@@ -3450,18 +3450,23 @@ workinggroups:
     - github: ahrtr
       name: Benjamin Wang
       company: VMware
+      email: benjamin.ahrtr@gmail.com
     - github: hakman
       name: Ciprian Hacman
       company: Microsoft
+      email: ciprianhacman@gmail.com
```
[the remark this answers is not in the document]
Comment (ahrtr) [reply sigs.yaml]: updated.

**Comment by ahrtr** [comment]
/hold cancel

All confirmed now.

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
thanks!

**Pull request merged**